.github/workflows/main.yml

```diff
@@ -16,19 +16,20 @@ jobs:
       matrix:
         node-version:
           - 20
-          - 16
+          - 18
         os:
           - ubuntu-latest
           - windows-latest
-        include:
-          - os: ubuntu-latest
-            node-version: 18
+        # Even numbers of node-version
+        # include:
+        #   - os: ubuntu-latest
+        #     node-version: 18
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
+      - run: npm install --legacy-peer-deps
```

Comment:
This PR is ready for review, but we should either keep this flag or wait for  https://github.com/babel/babel/issues/16220